examples/gno.land/p/demo/mux/router.gno

```diff
@@ -32,6 +32,9 @@ func (r *Router) Render(reqPath string) string {
 			patPart := patParts[i]
 			reqPart := reqParts[i]
 
+			if strings.Contains(patPart, "?") {
```

Comment:
Alternatively I think you could also use the following instead of `strings.Contains()` if you want to make it explicit that you want just to check that a part starts with "?":

 ```go
if strings.HasPrefix(patPart, "?") {
```

Also, would it make sense to keep iterating parts when the first "?" is found? Because if so it might be possible to `break` the loop instead of `continue`. I wonder if continuing looping the query arguments might potentially lead to issues 🤔